src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from '../../../../common';
+
+import { LOCALES, SHOW_COURSE_BTN_TXT } from '../../../../constants';
+
+import {
+	formatCreationDate,
+	getCourseAuthor,
+	getCourseDuration,
+} from '../../../../helpers';
+import { Card, Col, Row } from 'react-bootstrap';
+
+const CourseCard = ({ course, authors }) => {
+	const { title, description, duration, creationDate } = course;
+
+	const authorsFormatted = getCourseAuthor({ course, authors }).join(', ');
+	const durationFormatted = getCourseDuration(duration);
+	const creationDateFormatted = formatCreationDate(creationDate, LOCALES.pl);
+
+	return (
+		<Card border='success' className='my-4'>
+			<Card.Body>
+				<Row>
+					<Col>
+						<Card.Title className='display-6 fw-bold'>{title}</Card.Title>
+						<Card.Text>{description}</Card.Text>
+					</Col>
+					<Col>
+						<Card.Text>
+							<span className='fw-bold'>Authors</span>: {authorsFormatted}
+						</Card.Text>
+						<Card.Text>
+							<span className='fw-bold'>Duration</span>: {durationFormatted}
+						</Card.Text>
+						<Card.Text>
+							<span className='fw-bold'>Created</span>: {creationDateFormatted}
+						</Card.Text>
```

Comment:
It does look like it can be implemented in a more extendable way.
Also, please move 'Created', 'Duration' and 'Authors' to the constants file.

```suggestion
	const cardItems = [
		{
			label: 'Authors',
			value: getCourseAuthor({ course, authors }).join(', '),
		},
		{
			label: 'Duration',
			value: getCourseDuration(duration),
		},
		{
			label: 'Created',
			value: 'formatCreationDate(creationDate, LOCALES.pl)',
		},
	];

	return (
		<Card border='success' className='my-4'>
			<Card.Body>
				<Row>
					<Col>
						<Card.Title className='display-6 fw-bold'>{title}</Card.Title>
						<Card.Text>{description}</Card.Text>
					</Col>
					<Col>
						{cardItems.map(({ label, value }) => (
							<Card.Text key={label}>
								<span className='fw-bold'>{label}</span>: {value}
							</Card.Text>
						))}
```